components/network/samba/patches/10-avahi.patch

```diff
@@ -1,20 +0,0 @@
---- samba-4.9.5/source3/smbd/avahi_register.c	2019-03-12 09:19:04.000000000 +0000
-+++ samba-4.9.5/source3/smbd/avahi_register.c	2019-03-18 17:58:47.199335371 +0000
-@@ -186,7 +186,7 @@
```

Comment:
What has happened to this patch?

Reply:
https://github.com/OpenIndiana/oi-userland/pull/5377